package.json

```diff
@@ -2,6 +2,8 @@
   "name": "@stacks/wallet-web",
   "description": "The Hiro Wallet is browser extension for interacting with Stacks apps.",
   "private": true,
+  "version": "2.20.1-dev.8",
```

Comment:
Shouldn't this '-dev' be removed ?

Reply:
No, it'll be auto-updated by the semantic release job

Reply:
Ok got it !